src/declarations/stencil-public-compiler.ts

```diff
@@ -273,11 +273,12 @@ export interface ConfigExtras {
    */
   cssVarsShim?: boolean;
 
+  // TODO(STENCIL-661): Remove code related to the dynamic import shim
   /**
    * Dynamic `import()` shim. This is only needed for Edge 18 and below, and Firefox 67
    * and below. Defaults to `false`.
    */
```

Comment:
Can we add a [`@deprecated` JSDoc tag](https://jsdoc.app/tags-deprecated.html) to the public compiler interface here? 
```suggestion
   * @deprecated Since Stencil v3.0.0. IE 11, Edge <= 18, and old Safari versions are no longer supported.
   */
```


That way Intellisense(s) can pick this up in editor (although the name should be _pretty_ self explanatory)


![Screenshot 2022-12-09 at 9 25 43 AM](https://user-images.githubusercontent.com/1930213/206723776-fc1c6bf0-c565-49df-9a81-92f77a7fbcc4.png)